src/librustdoc/html/render/mod.rs

```diff
@@ -170,6 +170,10 @@ crate struct SharedContext<'tcx> {
 }
 
 impl<'tcx> Context<'tcx> {
+    fn depth(&self) -> usize {
```

Comment:
I can add docs here if you like. Let me know what you think.

Reply:
Sure, that seems useful.